src/Containers/EditControlPanel/EditControlPanel.tsx

```diff
@@ -0,0 +1,160 @@
+import React, {useState} from 'react';
+import styled from "styled-components";
+import Button from "../../Inputs/Button/Button"
+
+export interface IEditControlPanel {
+    /**
+     * The table number for the table
+     */
+    TableNumber: string;
+    /**
+     * The number of seats at the table
+     */
+    NumberOfSeats: number;
+}
+
+/**
+ * Primary UI component for user interaction
+ * Edit control panel for the right side of the edit page
+ */
+export const EditControlPanel: React.FC<IEditControlPanel> = ({
+    TableNumber= 'T1',
+    NumberOfSeats = 1,
```

Comment:
where is onDeleteClick, onRotateClick etc Your missing the listeners for the button

Reply:
I didn't add these yet because I wanted to add everything to the page first. Otherwise I would create an empty function for the onClick methods. Will empty functions be ok?

Reply:
Yes Ofcourse becuase you are building the extensible component